src/relayclient/ContractInteractor.ts

```diff
@@ -28,8 +28,8 @@ let IRelayHubContract: Contract<IRelayHubInstance>
 let IForwarderContract: Contract<ITrustedForwarderInstance>
 
 export default class ContractInteractor {
-  private readonly web3: Web3
-  private readonly provider: provider
+  readonly web3: Web3
+  readonly provider: provider
```

Comment:
I don't think this is a good idea. Better create an API for the methods you want from the contracts interactor